crates/matrix-sdk/src/room/timeline/event_item.rs

```diff
@@ -404,58 +403,34 @@ impl From<RoomEncryptedEventContent> for EncryptedMessage {
     }
 }
 
-#[derive(Clone, Debug)]
+#[derive(Clone, Debug, Default)]
 pub struct BundledReactions {
-    /// Whether all reactions are known, or some may be missing.
-    ///
-    /// If this is `false`, the remaining reactions can be fetched via **TODO**.
-    pub complete: bool, // FIXME: Unclear whether this is needed
-
-    /// The reactions.
+    /// The reactions grouped by key.
     ///
     /// Key: The reaction, usually an emoji.\
-    /// Value: The count.
-    pub bundled: IndexMap<String, ReactionDetails>,
+    /// Value: The group of reactions.
+    pub bundled: IndexMap<String, ReactionGroup>,
```

Comment:
If this struct has only a single field, maybe it isn't needed at all? Could be a type alias `type BundledReactions = IndexMap<String, ReactionGroup>;` instead.

Reply:
You're right. I've been hesitating in case we need more fields in the future (e.g. when server-side aggregation is more usable), but we can always change it later.